<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite thread-count="10" parallel="tests" name="Failed suite [login]">
  <test thread-count="10" parallel="tests" name="Test.Ancera.Login1(failed)">
    <classes>
      <class name="Test.Ancera.Login.TestLogin">
        <methods>
          <include name="login"/>
          <include name="logout"/>
          <include name="endreport"/>
          <include name="extent"/>
          <include name="VerifyLogout"/>
        </methods>
      </class> <!-- Test.Ancera.Login.TestLogin -->
    </classes>
  </test> <!-- Test.Ancera.Login1(failed) -->
  <test thread-count="10" parallel="tests" name="Test.Ancera.Login2(failed)">
    <classes>
      <class name="Test.Ancera.Login.ForgotPassword">
        <methods>
          <include name="endreport"/>
          <include name="extent"/>
          <include name="VerifyLogin"/>
        </methods>
      </class> <!-- Test.Ancera.Login.ForgotPassword -->
    </classes>
  </test> <!-- Test.Ancera.Login2(failed) -->
</suite> <!-- Failed suite [login] -->
